app/mailers/pay_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class PayMailer < Pay::UserMailer
+  def subscription_renewing
+    @business = params[:pay_customer].owner.business
+    @renewal_date = params[:date].to_date
+
+    mail(
```

Comment:
Let's copy over (or ideally, reference) the defaults from `ApplicationMailer` here.

Reply:
I can use class-level `default` statements here, but that would probably override some settings in `pay.rb` for all pay emails, not just the `subscription_renewing` one. That's the reason why I've tried to avoid that.

Here are the original classes our custom mailer inherits from: [Pay::UserMailer](https://github.com/pay-rails/pay/blob/master/app/mailers/pay/user_mailer.rb), [Pay::ApplicationMailer](https://github.com/pay-rails/pay/blob/master/app/mailers/pay/application_mailer.rb).

So I'm just making sure - if you really want me to use the class-level `default` statements instead, let me know.